desktop/apps/auction2/queries/sale.js

```diff
@@ -2,6 +2,22 @@ export default function SaleQuery(id) {
   return `{
     sale(id: "${id}") {
       _id
+      associated_sale {
+        cover_image {
+          cropped(width: 260, height: 110) {
+            url
+          }
+        }
+        end_at
+        href
+        id
+        is_closed
+        is_live_open
+        is_preview
+        live_start_at
+        name
+        start_at
+      }
```

Comment:
Isn't the standard practice to co-locate a sufficiently renderable metaphysics query alongside the `<Component />` even if its not directly used within the code? So that when using it elsewhere you can import and compose it as a string similar to what we did with `current_auctions` in `artwork/components/client.coffee`?